tests/api2/test_disk_wipe.py

```diff
@@ -88,6 +90,14 @@ def test_disk_get_partitions_quick(dev_name):
 
 
 def test_disk_wipe_abort():
+    """Test that we can sucessfully abort a disk.wipe job"""
+    expected_pids = set()
+    if ha:
+        # In HA systems fenced may be using the disk.  Obtain the PID
+        # so that we can ignore it.
+        expected_pids.add(ssh('cat /run/middleware/fenced.pid'))
```

Comment:
I have a method for fenced to return the pid and whether or not it's running. failover.fenced.run_info